applications/vessel-history/src/features/profile/components/activity/Activity.tsx

```diff
@@ -64,7 +65,7 @@ const Activity: React.FC<ActivityProps> = (props): React.ReactElement => {
     },
     [dispatch, props, setMapCoordinates]
   )
-
+  const hasGFWEvents = useSelector(selectVesselId)
```

Comment:
I'd change the name of the variable with something more meaningful, maybe something like:
```suggestion
  const isGFWVessel = useSelector(selectVesselId)
```